csm/core/blogic/const.py

```diff
@@ -935,11 +935,16 @@
 CREATE_USER_OPERATION = 'CREATE_USER'
 GET_USER_OPERATION = 'GET_USER'
 DELETE_USER_OPERATION = 'DELETE_USER'
-RGW_CLIENT_ERROR_MSG = 'Can not process the request'
+MODIFY_USER_OPERATION = 'MODIFY_USER'
+RGW_CLIENT_ERROR_MSG = 'Unable to process the request'
 CREATE_KEY_OPERATION = 'CREATE_KEY'
 REMOVE_KEY_OPERATION = 'REMOVE_KEY'
-
-# RGW AdminOps- create-user
+RGW_CLIENT_ERROR_CODES = {
```

Comment:
RGW_CLIENT_ERROR_CODES -> S3_CLIENT_ERROR_CODES

Reply:
This will be taken care in -  https://github.com/Seagate/cortx-manager/pull/779